src/PKSim.Presentation/Presenters/Charts/IndividualSimulationComparisonPresenter.cs

```diff
@@ -66,10 +78,22 @@ public bool AnyCurves()
       protected override void OnDragDrop(object sender, IDragEvent e)
       {
          var droppedNodes = e.Data<IReadOnlyList<ITreeNode>>();
-         if (containsIndividualSimulationNodes(droppedNodes))
-            individualSimulationsFrom(droppedNodes).Each(addSimulationToChart);
-         else
+         var messages = new List<string>();
+         if (!containsIndividualSimulationNodes(droppedNodes))
+         {
             base.OnDragDrop(sender, e);
+            return;
+         }
+
+         individualSimulationsFrom(droppedNodes).Each(s =>
```

Comment:
@Yuri05 adding it to a message and showing the error. It was their all the time but the exception was swallowed lol